### PR TITLE
Updated Tizen Studio MD Fiels

### PR DESCRIPTION
**Pull request by junmoy**
Updated Tizen Studio MD Fiels: Common Tools, Download, Native-toolsm, Web-Tools, Covered-Page.md and index.md

### CODE REVIEW

File: docs/tizen-studio/common-tools/device-manager.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+The Device Manager provides a graphical user interface that is more convenient than the CLI (Command Line Interface). You can use it to connect and disconnect devices to and from your computer, as well as to access devices' internal file systems and push or pull files between the device and your computer. The Device Manager provides the device logs in a table format, and you can use filters to find the entries that interest you.
+
+To use the Device Manager with a specific device, you must connect the device over SDB (Smart Development Bridge). Physical devices can be connected with a USB cable, and emulators can be connected through the Emulator Manager. When the device is connected over SDB, it is automatically displayed in the Device Manager and its directory structure and logs are shown in the **File Explorer** and **Log** views, respectively.
+
+**Note** If you do not have the Device Manager installed, open the [Package Manager](../download/updating_sdk.htm), go to the **Main SDK** tab, and locate the **Device Manager** under **Tizen SDK Tools > Baseline SDK**. To install the tool, click **Install**.
+
+
+Accessing Device Manager {#access}
```
Comment (journey2w) [inline docs/tizen-studio/common-tools/device-manager.md]: I think you have to change the following:
From:
Accessing Device Manager {#access}

To:
<a name="access"></a>
## Accessing Device Manager 

There are this kinds of changes. Please update and resend a new pull request.

File: docs/tizen-studio/download/prerequisites.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ The following table lists the CPU, screen resolution, graphic card, driver, and
...
 | Screen resolution                         <td colspan=3>  Recommended: 1280 x 1024 |                                          |               |                                          |
-| Graphic card                             <td colspan=3>  Recommended: The following requirements have passed tests with the emulator.<br/><br/> **Supported graphic cards**<br/> - NVIDIA: NVIDIA® GeForce® 8300 GS, GeForce® 8500 GT, GeForce® GT 220, GeForce® GT 430, GeForce® GT 530, GeForce® GT 330M, GeForce® GTX 550Ti, NVIDIA® Quadro® NVS 290<br/> - ATI: RADEON HD 4850, RADEON HD 5450<br/> - Intel: HD Graphics 2000, HD Graphics 2500, HD Graphics 4000<br/><br/>  **Note**<br/>If the host machine is using the NVIDIA® Optimus® technology, the emulator works with the on-board graphics card. To prevent this, either disable the Optimus® technology, or set the emulator to run with the external NVIDIA graphics card.Integrated graphic cards inside Intel's Q33/Q35/Q43/Q45 motherboards are not supported.First generation Intel HD Graphics is not supported. |                                          |               |                                          |
```
Comment (journey2w) [inline docs/tizen-studio/download/prerequisites.md]: The column is broken. Please see this file in browser and fix it.

File: docs/tizen-studio/download/prerequisites.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ The following table lists the CPU, screen resolution, graphic card, driver, and
...
+| Graphic card                             <td colspan=3>  Recommended: The following requirements have passed tests with the emulator.<br/><br/> **Supported graphic cards**<br/> - NVIDIA: NVIDIA® GeForce® 8300 GS, GeForce® 8500 GT, GeForce® GT 220, GeForce® GT 430, GeForce® GT 530, GeForce® GT 330M, GeForce® GTX 550Ti, NVIDIA® Quadro® NVS 290<br/> - ATI: RADEON HD 4850, RADEON HD 5450<br/> - Intel: HD Graphics 2000, HD Graphics 2500, HD Graphics 4000<br/><br/>  > **Note**<br/>  > If the host machine is using the NVIDIA® Optimus® technology, the emulator works with the on-board graphics card. To prevent this, either disable the Optimus® technology, or set the emulator to run with the external NVIDIA graphics card.Integrated graphic cards inside Intel's Q33/Q35/Q43/Q45 motherboards are not supported.First generation Intel HD Graphics is not supported. |                                          |               |                                          |
```
Comment (journey2w) [inline docs/tizen-studio/download/prerequisites.md]: The column is broken. Please see this file in browser and fix it.

**Comment by mijin85cho** [comment]
This request is merged by #95, so I close this request

**Pull request closed without merging**